application/org.openjdk.jmc.ide.launch/src/main/java/org/openjdk/jmc/ide/launch/JfrLaunchPage.java

```diff
@@ -119,6 +121,22 @@ public void widgetSelected(SelectionEvent e) {
 		autoOpenCheckbox.setLayoutData(gd);
 	}
 
+	private void createLessThenOracleJdk11(Composite parent, int cols) {
+		lessThenOracleJDK11Checkbox = new Button(parent, SWT.CHECK);
```

Comment:
Here we're letting the user explicitly toggle this if it's a particular Oracle JDK, could `ConnectionToolkit.isOracle()` help here to automate this?

Reply:
the issue is eclipse here and due to it there is hardcoded default jvm. I've though that VMRunner from eclipse (it's used) could be the option but sadly not.